src/gameplay/glicko2.ts

```diff
@@ -0,0 +1,136 @@
+// @ts-nocheck
+
+const computeG = (phi) => {
+  return 1 / Math.sqrt(1 + (3 * phi ** 2) / Math.PI ** 2);
+};
+
+const computeE = (mu, mu_j, phi_j) => {
+  return 1 / (1 + Math.exp(-1 * computeG(phi_j) * (mu - mu_j)));
+};
+
+
+const computeNewVolatility = (delta, phi, v, sigma, tau) => {
```

Comment:
Add return type

Reply:
You can also collect the delta, phi, v, sigma and tau variables into a single class or type so that it'll be easier to manage.